README.md

```diff
@@ -21,4 +21,42 @@ Things you may want to cover:
 
 * Deployment instructions
 
-* ...
+# Installation
+
+1. Setup PostgreSQL
+
+  https://www.digitalocean.com/community/tutorials/postgresql-ubuntu-16-04-ru
+
+2. Install gem pg:
+
+  ```sh
+  # Install gem pg
+  $ sudo apt-get install libpq-dev
+  $ gem install pg  --   --with-pg-lib=/usr/lib
+  ```
+3. Install yarn
+
+  https://yarnpkg.com/lang/en/docs/install/#debian-stable
+
+  ```sh
+  $ yarn install --check-files
+  ```
+
+4. Then, in your project directory:
+
+  ```sh
+  $ bundle install
+  $ rake db:create
+  $ rake db:migrate
+  ```
+
+5. Run application: 
+
+  ```sh
+  $ foreman start -f Procfile.dev -p 3000
+  ```
+# Compatibility
+
+* Ruby 2.6.3
+* Rails 5.2.3
+* PostgreSQL 11.4
```

Comment:
[PostgreSQL 9.0 or later](https://bitbucket.org/ged/ruby-pg/wiki/Home)
I think such strong dependency version will be difficult to deploy for example, so it will be easier to be more flexible.